connector/signaltometricsconnector/config/config.go

```diff
@@ -0,0 +1,10 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package config // import "github.com/open-telemetry/opentelemetry-collector-contrib/connector/signaltometricsconnector/config"
+
+type Config struct{}
```

Comment:
Sounds good! I have added the config and basic validation but I have not added defaults and in-depth validation because they might lead to discussions and has a potential to bloat this PR. Let me know if you would prefer to add the whole configuration logic in this PR itself.

Reply:
Sure! That sounds good, will add a README with the configurations to get started. I also plan to add some example configurations and corresponding output but it doesn't make sense to add them now without the actual implementation bit...